scp/cc/cpio/client_providers/private_key_client_provider/src/private_key_client_provider.cc

```diff
@@ -59,8 +58,7 @@ ExecutionResult PrivateKeyClientProvider::Init() noexcept {
       private_key_client_options_->primary_private_key_vending_endpoint);
   for (const auto& endpoint :
        private_key_client_options_->secondary_private_key_vending_endpoints) {
-    if (!endpoint.private_key_vending_service_endpoint.empty())
-      endpoint_list_.push_back(endpoint);
```

Comment:
What about this?

Reply:
When we last discussed, I was told this was in the merge (or a PR) by Dom, but if it isn't this could be the change that's triggering the failure.